bowling/inc/lane.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+class Lane {
+public:
+    Lane(const std::string& name) : name_(name) {}
+
+    size_t getPlayersNum() const { return players_.size(); }
+    std::string getPlayer(size_t index) const { return players_.at(index); }
```

Comment:
`.at` can throw exception if index is out of range

Please take a look at my comment https://github.com/kszytko/testing/pull/13#discussion_r481389052

Reply:
done